.github/ISSUE_TEMPLATE/use-case.yml

```diff
@@ -0,0 +1,25 @@
+name: FAIR Mappings Use Case submission
+description: Use this template to submit a FAIR Mappings Use Case for the RDA FAIR Mappings Working Group.
+title: FAIR Mappings Use Case submission
+labels: [ use-case ]
+assignees: jkesanie
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Some instructions for the submission.
```

Comment:
I just note (as a reminder) that this will need to be replaced by actual instructions at some point.